### PR TITLE
Update the display of bundles.

### PR DESCRIPTION
**Pull request by huwshimi**
Correctly display the bundle details header.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- visit `/:flags:/react/`
- do a search for "django"
- scroll down and click on the first bundle result "django"
- check that the header looks OK and includes the default bundle icon

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3234/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3235/
Test PASSed.

File: jujugui/static/gui/src/app/components/entity-details/entity-details.js [hunk 1]
```diff
@@ -92,6 +92,37 @@ YUI.add('entity-details', function() {
...
+    },
+
+    /**
+      Get the element for the series.
+
+      @method _getSeries
+      @returns {String} The icon URL
+    */
+    _getSeries: function(series) {
+      console.log(series);
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-details/entity-details.js]: left over console.log

File: jujugui/static/gui/src/app/components/entity-details/entity-details.js [hunk 2]
```diff
@@ -92,6 +92,37 @@ YUI.add('entity-details', function() {
...
+      if (series) {
+        return <li className="header__series">{series}</li>;
+      } else {
+        return null;
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-details/entity-details.js]: Does null have any special functionality vs `undefined` as if you didn't explicitly return?

You can drop the `else` though

**Comment by hatched** [comment]
:+1: With trivials
QA :ok:

**Comment by makyo** [comment]
:+1:

File: jujugui/static/gui/src/app/components/entity-details/entity-details.js (hunk 2 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/entity-details/entity-details.js]: Nope, looks like I don't even need to return.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3246/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3247/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks @hatched and @makyo :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**